rocketpy/AeroSurface.py

```diff
@@ -202,15 +211,27 @@ def kind(self):
 
     @kind.setter
     def kind(self, value):
```

Comment:
Are we allowed to do so many things inside the `kind` setter? 
What other possibilites could we set instead?

Reply:
Maybe we could just set up the type and nosecone equation. 
And then create an function to do the rest?

Reply:
I think the "evaluate_geometrical_parameters" should be the best place for what we have now in the kind.setter method.
Seems like an item for a to-do list in future PRs.